climakitae/uncertain_utils.py

```diff
@@ -420,3 +530,55 @@ def compute_vmin_vmax(da_min, da_max):
     else:
         sopt = None
     return vmin, vmax, sopt
+
+
+def get_ks_pval_df(sample1, sample2, sig_lvl=0.05):
+    """Performs a Kolmogorov-Smirnov test at all lat, lon points
+
+    Parameters
+    ----------
+    sample1: xr.Dataset
+        first sample for comparison
+    sample2: xr.Dataset
+        sample against which to compare sample1
+    sig_lvl: Float
+        alpha level for statistical significance
+
+    Returns
+    -------
+    pandas.dataframe
```

Comment:
Needs to exactly match the object type, including capitalizations. Should be `pd.DataFrame`